lisa/tools/xfstests.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import PurePath
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Debian, Fedora, Posix
+
+from .git import Git
+from .make import Make
+
+
+class Xfstests(Tool):
+    repo = "https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git"
```

Comment:
can we specify a tag or version? It would be more stable than default branch.

Reply:
I checked that, xfstests doesn't have such stable tag/branch.
https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git
https://github.com/kdave/xfstests